silkrpc/commands/eth_util_api.cpp

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
I would like to have `commands` folder dedicated just to JSON RPC API implementation modules, so I propose to move this module in `core` or `core/rawdb` as `cached_accessors.hpp` or something similar. IMHO we should also refactor `accessors.hpp` module from functions to objects (it was my fault here, sorry) and then do the same also for `cached_accessors.hpp` but this is for another PR.